indexer/lib/handle-provider-events.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Handles the /provider-registered webhook
+ *
+ * @param {Env} env
+ * @param {string} provider
+ * @param {string} pdpUrl
+ * @returns {Promise<Response>}
+ */
+export async function handleProviderRegistered(env, provider, pdpUrl) {
+  if (
```

Comment:
We also validate if `provider` is a valid eth address (at least valid format of eth address) and that `pdpUrl` is valid a http/https URL.

Reply:
I propose to validate if `provider` is valid eth address and `pdpUrl` is valid url.

Reply:
Yeah, I think it's a great idea to add the validation here. When the validation fails, we need to alert us on Slack. 

- In this pull request, it's enough to print a structured log message via console.error.
- Then edit the description of the following issue and add another todo-item to create an alert for this place: https://github.com/filcdn/roadmap/issues/23